tasks/pkg-suse.yml

```diff
@@ -27,30 +27,55 @@
   when: datadog_agent_major_version|int < 7 and not ansible_check_mode
 
 - block:  # Work around due to SNI check for SLES11
-  - name: Stat if new RPM key already exists
+  - name: Stat if E09422B3 key (Expires 2022) RPM key already exists
     stat:
       path: /tmp/DATADOG_RPM_KEY_E09422B3.public
     register: ddnewkey
-  - name: Download new RPM key (SLES11)
+  - name: Download E09422B3 key (Expires 2022) RPM key (SLES11)
     get_url:
       url: "{{ datadog_zypper_gpgkey_e09422b3 }}"
       dest: /tmp/DATADOG_RPM_KEY_E09422B3.public
     when: not ddnewkey.stat.exists
   when: ansible_distribution_version|int == 11
 
-- name: Download new RPM key
+- name: Download E09422B3 key (Expires 2022) RPM key
   get_url:
     url: "{{ datadog_zypper_gpgkey_e09422b3 }}"
     dest: /tmp/DATADOG_RPM_KEY_E09422B3.public
     checksum: "sha256:{{ datadog_zypper_gpgkey_e09422b3_sha256sum }}"
   when: ansible_distribution_version|int >= 12
 
-- name: Import new RPM key
+- name: Import E09422B3 key (Expires 2022) RPM key
   rpm_key:
     key: /tmp/DATADOG_RPM_KEY_E09422B3.public
     state: present
   when: not ansible_check_mode
 
+- block:  # Work around due to SNI check for SLES11
+  - name: Stat if 20200908 key (Expires 2024) RPM key already exists
+    stat:
+      path: /tmp/DATADOG_RPM_KEY_20200908.public
+    register: ddnewkey
```

Comment:
Is it fine to register `ddnewkey` twice (once for the E09422B3 key, once for 20200908)? If it's being overridden, that should be okay (but we need an SLES 11 box to test).

We can also decide to use another variable name (eg. `ddkey_20200908`) to be sure there won't be any conflict.